gensim/models/word2vec.py

```diff
@@ -1466,7 +1466,7 @@ def load(cls, *args, **kwargs):
         for v in model.vocab.values():
             if hasattr(v, 'sample_int'):
                 break  # already 0.12.0+ style int probabilities
-            else:
+            elif hasattr(v, 'sample_probability'):
                 v.sample_int = int(round(v.sample_probability * 2**32))
```

Comment:
Earlier versions used `sample_probability` (a float) instead of `sample_int` for this purpose; this allows models saved from those versions to load properly.